### ISSUE TITLE
Ring a bell when a response is sent back 

### ISSUE THREAD

**Issue by zmaril** [deiucanta/chatpad#28]
Hello, 

I was wondering if you could have a setting to have chatpad ring a bell or otherwise make a noise whenever a response is sent back. With some of the API's, the responses are slow enough that I lose focus waiting. I'd like a notification of some sort, and not just a system one. 

Chatpad is great though! Very glad I don't have to build something like this myself.

**Comment by glinkot** [comment]
Good idea!  A little ding, particularly for GPT4 that takes a while, would be a nice addition.
